### PR TITLE
Fix length of null array bug

### PR DESCRIPTION
**Pull request by joextodd**
Who: @rajaishwary
Why: Crashes on a failed decode
What:

Check if data is null before attempting to unpack into a WritableArray

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -73,7 +73,7 @@ export default class App extends Component<{}> {
     this.onReceived = ChirpConnectEmitter.addListener(
       'onReceived',
       (event) => {
-        if (event.data) {
+        if (event.data.length) {
```
Comment (dinuchiriac) [inline App.js]: This will crash if data is null because `null.length` is not valid

Comment (dinuchiriac) [reply App.js]:
```
data = null
null
data.length
VM201:1 Uncaught TypeError: Cannot read property 'length' of null
    at <anonymous>:1:6
(anonymous) @ VM201:1
```

Comment (joextodd) [reply App.js]: The iOS / Android wrappers will always send an array back to JavaScript. It will just be empty if it was null in Android.

Comment (dinuchiriac) [reply App.js]: I'll leave it for you to decide but I think the wrapper code can change in feature and a simple check here will be usefull. 👍

**Pull request merged**